### ISSUE TITLE
Boot from flash

### ISSUE THREAD

**Issue by Jandro20** [tq-systems/MBa117xL.SW.MCU.BSP#1]
Hello

I'm currently working with [TQMa117xL](https://www.tq-group.com/en/products/tq-embedded/arm-architecture/tqma117xl/) with this repository. Already, we have a demo application with the default configuration explained in [README.md](https://github.com/tq-systems/MBa117xL.SW.MCU.BSP/blob/main/README.md). We can generate the binary with debug+ram and debug with [Ozone debugger - Segger](https://www.segger.com/products/development-tools/ozone-j-link-debugger/).

To flash the device, we recompile with debug+flash configuration and use [MCUXpresso Secure Provisioning Tool](https://www.nxp.com/design/design-center/software/development-software/mcuxpresso-software-and-tools-/mcuxpresso-secure-provisioning-tool:MCUXPRESSO-SECURE-PROVISIONING) with the necessary configuration to flash into TQMa117xL, we follow the normal process to upload the binary into flash, with burned fuses and boot mode needed. (This process is already tested with other binary generated with [MCUXpresso Integrated Development Environment (IDE)](https://www.nxp.com/design/design-center/software/development-software/mcuxpresso-software-and-tools-/mcuxpresso-integrated-development-environment-ide:MCUXpresso-IDE))

The normal process in Secure Provisioning return a SUCCESS result, but when we reboot the board with Internal Boot mode, the binary is not loaded and the board have not code loaded.

Is it needed an additional process to flash the board?

Thanks!

**Comment by tq-liedmann** [comment]
Hello Jandro, 

sorry for the delay, can you please contact us via the email address support@tq-group.com. 

Best regards

**Comment by tq-niebelm** [comment]
> Hello Jandro,
> 
> sorry for the delay, can you please contact us via the email address [support@tq-group.com](mailto:support@tq-group.com).
> 
> Best regards

@tq-liedmann Can we close this?

**Comment by tq-liedmann** [comment]
@tq-niebelm Yes, please close this request, the request has been handled via email.

**Comment by tq-niebelm** [comment]
> [@tq-niebelm](https://github.com/tq-niebelm) Yes, please close this request, the request has been handled via email.

Thanks
